### DISCUSSION TITLE
Can you integrate milvus with flutter?

### DISCUSSION THREAD

**Discussion by abcnow** [milvus-io/milvus#28743] [Ideas & Feature requests]
I wonder if there is any plugin for flutter/dart or any idea if there is something in the making for the near future?

**Comment by yhmo** [comment]
Milvus is an infrastructure database. Users can build their upper applications by using sdk or rest api to interact with milvus. 
Flutter is UI framework for mobile, I don't think there is any relationship between milvus and flutter. If you want to build a mobile app based on milvus, there should be a backend server for your mobile app. You can make this server by some frameworks such as flask, the server interacts with milvus by python sdk, and your mobile app interacts with the backend server by http requests.

**Comment by behradkhadem** [comment]
I'm currently working on milvus dart package if it is of any use
https://pub.dev/packages/milvus_dart
